### PR TITLE
Create Dell partner page and contact-form

### LINKED ISSUE #6008: [Dell partner page] - build page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created the /dell partner page
- Created the /dell#get-in-touch contact-us modal page
- Created the /dell/contact-us page
- Created the /dell/thank-you page
- Fixed a width in the /internet-of-things/appstore page

**NOTE** - *the reference architecture pages are not ready and the links hidden*

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/dell
    - http://0.0.0.0:8001/dell#get-in-touch (fill in the form)
    - http://0.0.0.0:8001/dell/contact-us (fill in the form)
    - http://0.0.0.0:8001/dell/thank-you (you need to manually go there as the contact us will take you to ubuntu.com)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare /dell to the [copy doc](https://docs.google.com/document/d/1ZZQdZt2HFiLwjYYGsb-q7Zncdj0kdv0RBxFm7cz7Gu8/edit#)
- Compare /dell#get-in-touch to the [copy doc](https://docs.google.com/document/d/1tv_fsVebbIPgKjwnmawF6hlWTTPkvjE2jAjH3MBp5vU/edit#heading=h.tk4y6g6el1tm)

## Issue / Card

Fixes #6008

### CODE REVIEW

File: templates/dell/index.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  <div class="row">
+    <div class="col-6">
+      <h2>
+        In the data centre
+      </h2>
+      <h3>
+        Dell servers provisioning with MAAS
+      </h3>
+      <p>
+        At the heart of the data centre and a key component to building a hybrid strategy, Dell PowerEdge servers come certified with Ubuntu as the operating system.  Offering the performance and versatility to get the most from your infrastructure, Dell’s flexible designs offer a range of options for price, functionality and scalability to meet your workload needs. Those servers can be provisioned with <a href="https://maas.io">MAAS</a> for a cloud-like experience and enhanced automation.
```
Comment (solazio) [inline templates/dell/index.html]:
```suggestion
        At the heart of the data centre and a key component to building a hybrid strategy, Dell PowerEdge servers come certified with Ubuntu as the operating system.  Offering the performance and versatility to get the most from your infrastructure, Dell’s flexible designs offer a range of options for price, functionality and scalability to meet your workload needs. Those servers can be provisioned with <a href="https://maas.io" class="p-link--external">MAAS</a> for a cloud-like experience and enhanced automation.
```

File: templates/dell/index.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  <div class="row">
+    <div class="col-6">
+      <h2>
+        At the edge
+      </h2>
+      <h3>
+        Redefine intelligence at the edge
+      </h3>
+      <p>
+        Edge computing is transforming all sectors from robotics to 5G infrastructure. Ubuntu is the standard for <a hrer="/internet-of-things">Linux for edge deployments</a>, bringing efficiencies with regular updates, robust security, and enhanced controls. Combine this with Dell Edge Gateways, that are engineered for the extreme environment and can withstand even the harshest of conditions, making them a premium gateway option.
```
Comment (solazio) [inline templates/dell/index.html]:
```suggestion
        Edge computing is transforming all sectors from robotics to 5G infrastructure. Ubuntu is the standard for <a href="/internet-of-things">Linux for edge deployments</a>, bringing efficiencies with regular updates, robust security, and enhanced controls. Combine this with Dell Edge Gateways, that are engineered for the extreme environment and can withstand even the harshest of conditions, making them a premium gateway option.
```

**Comment by solazio** [comment]
@pmahnke two QA comments on the /dell page:
- "Deploy containers at scale with Charmed Kubernetes" is missing `[See the Reference Architecture components]`
- "Enterprise clouds with OpenStack" is missing `[View the OpenStack cloud components]`

But they might not be required on the page.

File: templates/internet-of-things/appstore.html [hunk 3]
```diff
@@ -305,7 +305,7 @@ <h2>Ready to innovate? <br class="u-hide--small" />We would love to hear from yo
       </p>
     </div>
     <div class="col-5 u-align--center u-vertically-center">
-      <img src="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg" class="u-hide--small" width="300px" alt="">
+      <img src="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg" class="u-hide--small"  width="250" height="178" alt="">
```
Comment (solazio) [inline templates/internet-of-things/appstore.html]: Shall we use `height="auto"` as the width is already set by `width`

**Comment by anasereijo** [comment]
@pmahnke can you please amend the images sizes otherwise looks perfect 👍 🥇

**Comment by anasereijo** [comment]
Screenshot 2019-11-05 at 13 40 10
Screenshot 2019-11-05 at 13 38 11
Screenshot 2019-11-05 at 13 34 09

**Comment by pmahnke** [comment]
@anasereijo done
@solazio fixed, except your note on the missing reference arch's, still waiting for these

**Comment by anasereijo** [comment]
awesome

**Pull request merged**